fili-core/src/main/java/com/yahoo/bard/webservice/web/DataApiRequest.java

```diff
@@ -296,6 +303,55 @@ public DataApiRequest(
     }
 
     /**
+     * Update sort field string by truncating the dateTime if it is present.
+     *
+     * @param sorts  String of sort columns
+     *
+     * @return Updated sort column if dateTime exists. Otherwise return as it is received.
+     */
+    protected String truncateTimeSort(String sorts) {
+
+        if (sorts != null && !sorts.isEmpty() && sorts.contains(DATE_TIME_STRING)) {
+            if (sorts.contains(",")) {
+                //Truncate the dateTime sort column from the api sort string
+                return sorts.substring(sorts.indexOf(",") + 1, sorts.length());
```

Comment:
This isn't safe. It doesn't work correctly if a dimension exists that starts with dateTime but isn't exactly dateTime. For example, if I define a dimension called `dateTimeInOtherTz` and included it at the _end_ of my sorts string, this code would incorrectly truncate the first dimension from the sorts clause, even if it wasn't dateTime.